pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1810,55 +1810,50 @@ def test_dt64ser_sub_datetime_dtype(self):
     # TODO: This next block of tests came from tests.series.test_operators,
     # needs to be de-duplicated and parametrized over `box` classes
 
-    def test_operators_datetimelike_invalid(self, all_arithmetic_operators):
-        # these are all TypeEror ops
+    @pytest.mark.parametrize(
+        "left, right, op_fail",
+        [
+            [
+                [Timestamp("20111230"), Timestamp("20120101"), NaT],
+                [Timestamp("20111231"), Timestamp("20120102"), Timestamp("20120104")],
+                ["__sub__", "__rsub__"],
+            ],
+            [
+                [Timestamp("20111230"), Timestamp("20120101"), NaT],
+                [timedelta(minutes=5, seconds=3), timedelta(minutes=5, seconds=3), NaT],
+                ["__add__", "__radd__", "__sub__"],
+            ],  # TODO(jreback) __rsub__ should raise?
```

Comment:
> TODO(jreback) __rsub__ should raise?

IIUC this is referring to `Series([timestamps]).__rsub__(Series([timedeltas]))`, which yes, should raise